sh/configure_env.sh

```diff
@@ -161,16 +161,38 @@ LDARGS=$(echo "$LDARGS" | sed -e "s/ CoreFoundation//g")
 # IFS = Internal Field Separator - allows looping over lines with spaces
 IFS=$'\n'
 
-
+ANACONDA=0
+LIBLDARGS=""
 # Add library path if using anaconda python
 PYVER=$($PY --version 2>&1 )
 if [[ "$PYVER" == *"Anaconda"* ]]; then
+  ANACONDA=1
   ANACONDA_LIB_PATH=$(linkread $(dirname $(which $PY))/../lib)
-  LDARGS="-L${ANACONDA_LIB_PATH} -Wl,-rpath,${ANACONDA_LIB_PATH} $LDARGS -lmkl_rt -lpthread"
-  LDARGS="-dynamiclib -undefined dynamic_lookup -Wl,-headerpad_max_install_names"
-  CXXFLAGS="-DSCIPY_MKL_H -DHAVE_CBLAS -DTICK_CBLAS_AVAILABLE $CXXFLAGS"
+  if [[ "$unameOut" == "Darwin"* ]]; then
+    # Unknown reason why these are not required for Anaconda -
+    #  Not just that but neither is the -lpython
+    LDARGS="" #-L${ANACONDA_LIB_PATH} -Wl,-rpath,${ANACONDA_LIB_PATH} $LDARGS -lmkl_rt -lpthread"
+    LIBLDARGS="-dynamiclib -undefined dynamic_lookup -Wl,-headerpad_max_install_names"
+    CXXFLAGS="-DSCIPY_MKL_H -DHAVE_CBLAS -DTICK_CBLAS_AVAILABLE $CXXFLAGS"
+  fi
 fi
 
+MKN_P_ARRAY=(lib_name=$LIB_POSTFIX)
+case "${unameOut}" in
+    Linux*)     LDARGS="${LDARGS}";;
+    Darwin*)    LDARGS="${LDARGS}";;
+    CYGWIN*)    LDARGS="${LDARGS}";;
+    MINGW*)     LDARGS="${LDARGS}";;
+    *)          LDARGS="${LDARGS}";;
+esac
```

Comment:
does this case do anything more than `LDARGS="${LDARGS}"`?

Reply:
This is a place holder for the "MKN_P_ARRAY" in case we wish to specificy platform specific properties

Reply:
Oh, ok maybe it's worth leaving a tiny comment just before defining `MKN_P_ARRAY` :)

Reply:
sure